Assets/MixedRealityToolkit/MSBuild/MSBuildTemplates/WSA.Player.Template.props

```diff
@@ -0,0 +1,32 @@
+<Project>
+  <PropertyGroup>
+    <AssemblySearchPaths>$(AssemblySearchPaths);<!--PLATFORM_COMMON_ASSEMBLY_SEARCH_PATHS_TOKEN-->;C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETCore\v4.5;$(UnityEditorInstallFolder)Data\PlaybackEngines\MetroSupport\Managed\UAP\</AssemblySearchPaths>
```

Comment:
>C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETCore\v4.5 [](start = 99, length = 77)

These are things that are always installed to C: right (or the windows install drive?) Is there any risk that these might actually be on another drive letter?

Reply:
There is, and I am sure this won't work on Mac either. That, however, is P3 I would say, where P1 is get build working, P2 is local dev box.

Having said that, I may need to update it as we get build stuff.

---
In reply to: [301780755](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5215#discussion_r301780755) [](ancestors = 301780755)